homeassistant/components/reolink/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Reolink IP NVR/camera",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/reolink",
-  "requirements": ["reolink-aio==0.1.2"],
+  "requirements": ["reolink-aio==0.1.3"],
```

Comment:
Side note: If you're following semantic versioning, adding new features should normally bump the minor version specifier.

Reply:
True, I did that wrong, I will keep it in mind for the next bump